### PR TITLE
Update IDs for RazorComponents

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
CC @Eilon on this since he seems to have the context here.

### CODE REVIEW

**Comment by Eilon** [comment]
I actually have almost no context :smile: It's @phenning that's the expert!

**Review by phenning — APPROVED** [review]

**Comment by ryanbrandenburg** [comment]
@phenning doesn't have write access so his approval isn't sufficient. @ajaybhargavb, believe in me who believes in @phenning.

**Review by ajaybhargavb — APPROVED** [review]

**Pull request merged**
